packages/ui-components/src/components/HeaderContainer/HeaderContainer.component.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+
+const headerContainerStyles = `
+  jn-flex
+  jn-flex-col
+  jn-sticky
+  jn-top-0
+  jn-z-50
+  jn-shadow-lg
+  jn-bg-theme-global-bg
+`
+
+export const HeaderContainer = ({
+  fullWidth = false,
+  className = "",
+  children = null,
+  ...props
+}: HeaderContainerProps) => {
+  return (
+    <div
+      className={`
+    juno-header-container 
+    ${!fullWidth ? "jn-w-full 2xl:jn-container 2xl:jn-mx-auto" : ""}
```

Comment:
Can you confirm that this statement is correct. It looks like if not fullWidth then add `jn-w-full` class

Reply:
Yes, this is correct. `jn-w-full` (default) is active on smaller displays, on larger resolutions `jn-container` is active instead